src/main/webapp/app/overview/course-exercises/course-exercises.component.ts

```diff
@@ -17,16 +17,25 @@ import { CourseScoreCalculationService } from 'app/overview/course-score-calcula
 import { Exercise, ExerciseType } from 'app/entities/exercise.model';
 import { ExerciseService } from 'app/exercises/shared/exercise/exercise.service';
 
+/**
+ * enumerator for filtering exercise
```

Comment:
It would be good to add the @enum [type] and @readonly tag for enumerators and maybe describing the enum types in a more detailed way where it makes sense